index.php

```diff
@@ -58,8 +58,8 @@
 switch ($action) {
     case 'disable':
         if (in_array($factor, $enabledfactors)) {
-            tool_mfa_set_factor_config(array('enabled' => 0), 'factor_' . $factor);
-            tool_mfa_change_factor_order($factor, $action);
+            \tool_mfa\manager::set_factor_config(array('enabled' => 0), 'factor_' . $factor);
```

Comment:
change_factor_order() can accecpt a disable / enable, seems like we should just user that

Also seems like change_factor_order should be better named as do_factor_action or similar?